service/src/main/frontend/README.md

```diff
@@ -0,0 +1,41 @@
+# Quasar App (frontend)
+
+Frontend for Data Dictionary
+
+## Install the dependencies
+```bash
+yarn
+# or
+npm install
+```
+
+### Start the app in development mode (hot-code reloading, error reporting, etc.)
+```bash
+quasar dev
+```
+
+
+### Lint the files
+```bash
+yarn lint
+# or
+npm run lint
+```
+
+
+### Format the files
+```bash
+yarn format
+# or
+npm run format
+```
+
+
+
+### Build the app for production
```

Comment:
The app should be built by maven for production.  Perhaps, consider rephrasing this comment?

Reply:
I added a proper README to setup/run/test the project.